tests/components/switch/test_init.py

```diff
@@ -30,11 +33,13 @@ def tearDown(self):
 
     def test_methods(self):
         """Test is_on, turn_on, turn_off methods."""
-        assert setup_component(
-            self.hass, switch.DOMAIN, {switch.DOMAIN: {CONF_PLATFORM: "test"}}
-        )
+        assert setup_component(self.hass, switch.DOMAIN,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)